eth/executionclient/execution_client.go

```diff
@@ -268,3 +334,17 @@ func (ec *ExecutionClient) reconnect(ctx context.Context) {
 
 	logger.Info("reconnected")
 }
+
+// func spitToRanges(start uint64, stop uint64, n uint64) []struct{} {
```

Comment:
Do we need to keep this? Can we delete it?

Reply:
Yes, lets remove as it wasnt needed at the moment